### DISCUSSION TITLE
[Python/Flask] Error: Function Runtimes must have a valid version, for example `now-php@1.0.0`.

### DISCUSSION THREAD

**Discussion by duotangx** [vercel/community#6932] [Help]
### Summary

Receive the above error when attempting to migrate my builds configuration to a functions configuration and deploy to Vercel. Copy of my vercel.json file is below.

```
{
    "version": 2,
    "functions": {
        "api/**/*.py": {
            "runtime": "@vercel/python",
            "maxDuration": 300
        }
    },
    "routes": [
        {
            "src": "/",
            "dest": "/api/index.py"
        },
        {
            "src": "/transcript",
            "dest": "/api/transcript.py"
        },
        {
            "src": "/setup_db",
            "dest": "/api/setup_db.py"
        }
    ]
}
```

### Steps to Reproduce

Switch from builds configuration to functions configuration, splitting previous app.py file into multiple .py files and using app routes.

**Comment by will-holley** [comment]
Check https://github.com/orgs/vercel/discussions/6287 and see whether that helps.

[a comment by an automated account is not shown]
